ffi/build.py

```diff
@@ -134,9 +134,12 @@ def main_posix(kind, library_ext):
         print(msg)
         print(warning + '\n')
     else:
-        if not (out.startswith('8.0.') or out.startswith('7.0.')
+
+        if not (out.startswith('9.0.')
+                or out.startswith('8.0.')
+                or out.startswith('7.0.')
```

Comment:
If `llvmlite` continues to support 7 and 8, but in prod everything moves to 9, how do we know if 7 and 8 got accidentally broken by some incoming change? I think all supported platforms work on 9 so it's not like we've got a spread of LLVM possibilities running to track this. Perhaps this should just be a move to 9 or comment about 7 and 8 likely to work but out of support?

Reply:
Also think the docs and README will need a patch for whatever changes are determined?

Reply:
let's just move to llvm9. we won't be testing llvm 8, 7 anymore.

Reply:
Agree. There's a few references to 7 and 8 in the source that need patching.
```
docs/source/user-guide/ir/index.rst:<http://llvm.org/releases/7.0.0/docs/LangRef.html>`_.
docs/source/admin-guide/install.rst:#. Download the `LLVM 7.0.0 source code <http://releases.llvm.org/7.0.0/llvm-7.0.0.src.tar.xz>`_.
docs/source/admin-guide/install.rst:   or `LLVM 8.0.0 source code <http://releases.llvm.org/8.0.0/llvm-8.0.0.src.tar.xz>`_
docs/source/conf.py:    'llvm': ('http://llvm.org/releases/8.0.0/docs', None),
```